src/test/java/com/google/sps/servlets/ScheduledInterviewServletTest.java

```diff
@@ -182,6 +182,17 @@
               Instant.parse("2020-07-20T13:30:00Z"), Instant.parse("2020-07-20T13:45:00Z")),
           /*id=*/ -1,
           /*scheduled=*/ false);
+  private final Person shadow =
+      Person.create(
+          emailToId("shadow@gmail.com"),
+          "shadow@gmail.com",
+          "Shadow_first_name",
+          "",
```

Comment:
These empty strings would generally merit a parameter comment as well (as is done for okShadow).

Reply:
Changed these to non empty strings